src/Loadbalancers/ListenerClient.php

```diff
@@ -277,6 +277,21 @@ public function updateCert($id, Cert $cert)
                 return new Cert($this->apiToFriendly($response->data, self::CERT_MAP));
             });
     }
+    
+    public function updateBind($id, Bind $bind)
+    {
+        $response = $this->patch(
+            "v2/listeners/$id/binds/$bind->id",
```

Comment:
does this not need to be "v2/listeners/$id/binds/{$bind->id}" ?

Reply:
I thought so too but the updateCert function this is copied from doesn't and it works.

I can change it anyway, probably make it a bit clearer